src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp

```diff
@@ -415,15 +421,15 @@ void CompositionalMultiphaseWell::validateInjectionStreams( WellElementSubRegion
     {
       real64 const compFrac = injectionStream[ic];
       GEOS_THROW_IF( ( compFrac < 0.0 ) || ( compFrac > 1.0 ),
-                     "WellControls " << wellControls.getDataContext() <<
-                     ": Invalid injection stream for well " << subRegion.getName(),
+                     "WellControls " << wellControls.getName() <<
+                     "Invalid injection stream for well " << subRegion.getName(),
```

Comment:
```suggestion
                     "Invalid injection stream for well " << subRegion.getName(),
```

And same upper and below